index.js

```diff
@@ -152,30 +192,15 @@ function sendNotification(endpoint, params) {
 
       if (vapid && !isGCM) {
         // VAPID isn't supported by GCM.
+        vapid.audience = urlParts.protocol + '//' + urlParts.hostname;
```

Comment:
Thanks for the explanation @jrconlin.

In this case, this PR is a bad change and audience be defined externally from this library and it just so happens for FCM, the feed has to be for FCM and for mozilla, it can be the original url used. @marco-c does that sound good to you?